challenges/shift-array/test_shift-array.py

```diff
@@ -0,0 +1,7 @@
+import shift_array as sa
+
+
+def test_insertShiftArray():
+    assert sa.insertShiftArray([1, 2, 3, 4], 5) == [1, 2, 5, 3, 4]
+    assert sa.insertShiftArray([1, 2, 3, 4, 5, 6, 7, 8], 5) == [1, 2, 3, 4, 5, 5, 6, 7, 8]
+    assert sa.insertShiftArray([20, 30, 59, 70], 200) == [20, 30, 200, 59, 70]
```

Comment:
Instead of having all these assert statements in one test, you should make a separate test function for each statement. You also might want to have more of a variety for your tests. Every one of these is a list of even length, what about odd length? or an empty array?